packages/lesswrong/server/collections/reports/mutations.ts

```diff
@@ -27,26 +27,52 @@ async function postReportsToSunshine(report: DbReport, context: ResolverContext)
   ]);
   const reportedUser = await context.Users.findOne({ _id: report.reportedUserId ?? comment?.userId ?? post?.userId });
 
+  const contentSlug = report.commentId ? `a comment on ${post?.title}` : report.postId ? `the post ${post?.title}` : `user ${reportedUser?.displayName}`;
+  const description = report.description ?? '';
+  const userLink = user ? `https://${process.env.SITE_URL ?? 'lesswrong.com'}/users/${user.slug}` : '';
+  const userName = user?.displayName ?? '';
+  const url = `https://${process.env.SITE_URL ?? 'lesswrong.com'}${report.link}`;
```

Comment:
Is `SITE_URL` a new env var?  (We have `getSiteUrl`, which uses the `siteUrl` setting and defaults to lolcahost:3000 on dev.)

But my guess is that you probably actually just always want to link to prod here?

Reply:
(Also we have a `userGetProfileUrl` function, though admittedly that does use `getSiteUrl`, which means it'll link to localhost if you invoke it when running a local instance.)

Reply:
yeah I guess I'm not sure whether it's worth doing something different for AF?

Reply:
Oh, the env variable is supposed to be VERCEL_URL